### PR TITLE
test: Add query related test cases for default and null support

### PR DESCRIPTION
**Pull request by qixuan0212**
issue : #36129

### CODE REVIEW

**Comment by qixuan0212** [comment]
rerun go-sdk

**Comment by ShineYellow** [comment]
rerun go-sdk

File: tests/python_client/testcases/test_query.py [hunk 1]
```diff
@@ -4135,3 +4135,357 @@ def test_query_iterator_expr_empty_with_random_pk_pagination(self, batch_size, o
...
+        collection_w.load()
+        # 2. search iterator
+        expr = "int64 >= 0"
+        collection_w.query_iterator(batch_size, expr=expr,
+                                    check_task=CheckTasks.check_query_iterator,
+                                    check_items={"count": ct.default_nb,
+                                                 "batch_size": batch_size})
+
+    @pytest.mark.tags(CaseLabel.L1)
+    def test_query_normal_none_data_partition_key(self, enable_dynamic_field, null_data_percent):
```
Comment (binbinlv) [inline tests/python_client/testcases/test_query.py:4468]: This case may block the CI if issue #36213 is fixed, so here may add skip fixture.

Comment (qixuan0212) [reply tests/python_client/testcases/test_query.py:4469]: add skip done
